ninaddaudkhane-minneapolis.md

```diff
@@ -0,0 +1,7 @@
+First change in this file!!
+
+Places to eat:
+
+1) Spoon and Stable
+2) Houlihaans
+3) Natures Basket
```

Comment:
If you want to add places to visit, can you provide some awesome craft breweries, I've already been to Tin Whiskers in St. Paul 😄 🍺